packages/v4/src/content/design-guidelines/usage-and-behavior/toolbar/toolbar.md

```diff
@@ -9,147 +9,126 @@ Toolbars can be included to allow a user to manage and manipulate a data set. Da
+<img src="./img/toolbar-with-divider.png" alt="toolbar with divider" width="1211" />
 
+## Usage
+The PatternFly toolbar is designed to be very flexible and allow a variety of combinations of items and groups as outlined above. The following are some guidelines that should be followed in laying out your toolbar.
 
-**2. Filter group:** PatternFly supports a variety of common filter types that can be used alone or in combination to construct a custom filter. You may consider placing filters inside of a toggle group to make you toolbar responsive. See [Filters](/design-guidelines/usage-and-behavior/filters) for more information about available filters.
+* When used, bulk selection should be the left-most element.
+* Filters and global actions may follow.
+* When pagination is used, it should be the right-most element.
+* When pagination is not used, the item count should be the right-most element.
 
+### Examples
+**Basic toolbar**
+<img src="./img/basic-toolbar-items.png"  alt="toolbar with standard items"  width="1210"/>
 
-**3. Icon button group:** Actions or settings that are represented by icons can be added in an Icon button group. In the example, Sort and Export are represented as icons.
+The basic toolbar is sufficient for most cases. The example above shows the toolbar with default spacing. It includes the following elements:
+1. **Bulk selector:** The bulk selector supports selection of multiple items in a list, table, or card grid. It is always the leftmost item.
+2. **Filter:** The example shows an attribute-value filter, but any supported filter can be used here. See [Filters](/design-guidelines/usage-and-behavior/filters) usage guidelines for more information.
+3. **Overflow menu:** In this example an overflow menu is used to hold global actions. But a simple button group could also be used here if you have two or less actions.
+4. **Item count**: When a filter is applied, the item count will be updated to represent the filtered subset of items. For example, “10 of 17 items.” The item count should be the right-most element.
 
+**Toolbar with pagination**
+<img src="./img/basic-toolbar-pagination.png"  alt="basic toolbar with pagination"  width="1210"/>
 
-**4. Action button group:** Any global actions that apply to selected data items are included left-aligned following filter and icon buttons. These may be placed within an overflow menu component to collapse the group as the viewport shrinks. No more than two items should be exposed as buttons. Button labels should be limited in length to preserve horizontal space.
+When pagination is used, the pagination component replaces the item count. It should be the right-most element. Here, the toolbar is shown with both full and compact pagination options.
```

Comment:
I believe we decided to prescribe `compact pagination` always be used in header applications. Like, when a toolbar appears above a dataset.

Reply:
I think this guidance makes sense, although I would not want to say "always" as you could opt to use the full pagination as it applies to your use case. @gdoyle1 what do we currently default to in the Sketch template? Another area where we should try to be consistent.

Reply:
OK. I added a statement about this.